src/hash-object.js

```diff
@@ -40,7 +40,7 @@ function prepareObject(value) {
     }
   } else if (value instanceof FP_Type) {
     return value.toString();
-  } else if (typeof value === 'object') {
+  } else if (typeof value === 'object' && value !== null) {
```

Comment:
Looks like a fix for another issue.

Reply:
Perhaps so; I needed this for my tests of this feature to pass!

Reply:
I've added a test for it explicitly. We can break it out into a separate PR, but this one will depend on that one.

Reply:
Ah, I see https://github.com/HL7/fhirpath.js/pull/139 addresses it. If we land that I can remove the changes here.